runtime-display/libxnvctrl/spec

```diff
@@ -0,0 +1,3 @@
+VER=535.54.03
```

Comment:
Do we need to sync the versions with that of our `nvidia` package? cc @cthbleachbit

Reply:
Will have to test whether we need to bind versions

Reply:
Tested 470.63.01, works flawlessly with MangoHUD. I'm confident that the version does not need to keep the same with `nvidia`. Will now upgrade to the latest version [535.113.01](https://github.com/NVIDIA/nvidia-settings/releases/tag/535.113.01).